jenkins-agent/Dockerfile_build

```diff
@@ -0,0 +1,38 @@
+ARG version=latest
```

Comment:
do we need to provide this build arg?

Reply:
I'd say it's a reasonable default, even if it can be a bit dangerous in some situations :-)